app/HMS/Repositories/Doctrine/DoctrineUserRepository.php

```diff
@@ -40,20 +40,26 @@ public function findByEmail(string $email)
 
     /**
      * @param  string $searchQuery
+     * @param  bool $hasAccount limit to users with associated accounts
      * @return array
      */
-    public function searchLike(string $searchQuery)
+    public function searchLike(string $searchQuery, ?bool $hasAccount = false)
```

Comment:
I'm calling with `$this->userRepository->searchLike($searchQuery, $request->input('withAccount', false))`
and `$request->input` can return null